vendor/bat-native-ads/src/bat/ads/internal/database/tables/creative_ads_database_table.cc

```diff
@@ -136,6 +143,21 @@ void CreativeAds::MigrateToV10(DBTransaction* transaction) {
   CreateTableV10(transaction);
 }
 
+void CreativeAds::MigrateToV12(DBTransaction* transaction) {
```

Comment:
v10 code can be removed and we can just rebuild the table in v12 for catalog changees

Reply:
fixed